content/may2015.md

```diff
@@ -40,7 +40,7 @@ Transfer-Encoding: chunked
 
 Blank fields are omitted.
 
-All timestamps are returned in [FHIR<sup>®</sup> standard date/dateTime format](http://www.hl7.org/implement/standards/fhir/datatypes.html#date).
+All timestamps are returned in [FHIR<sup>®</sup> standard date/dateTime formats](http://www.hl7.org/implement/standards/fhir/datatypes.html#date).
```

Comment:
This link should probably be http://www.hl7.org/implement/standards/fhir/2015may/datatypes.html#date